controllers/apps/operations/reconfigure.go

```diff
@@ -315,3 +314,20 @@ func (r *reconfigureAction) Action(reqCtx intctrlutil.RequestCtx, cli client.Cli
 	resource.OpsRequest.SetStatusCondition(*condition)
 	return nil
 }
+
+func needReconcile(request *appsv1alpha1.OpsRequest) bool {
+	// Update params to configmap
+	if request.Spec.Type != appsv1alpha1.ReconfiguringType ||
```

Comment:
refact the naming of 'reconfiguring/reconfigure/config/configuration', better to reduct to 'reconfigure/config'